Makefile

```diff
@@ -68,9 +74,8 @@ set -e ;\
 TMP_DIR=$$(mktemp -d) ;\
 cd $$TMP_DIR ;\
 curl -Lo protoc.zip $(2) ;\
-unzip -q protoc.zip bin/protoc ;\
-cp bin/protoc $(1) ;\
-chmod a+x $(1)/protoc ;\
+unzip -q protoc.zip bin/protoc -d $(1)/. ;\
```

Comment:
what about this change 
```diff
diff --git a/Makefile b/Makefile
index 367fb37..7226e6f 100644
--- a/Makefile
+++ b/Makefile
@@ -68,8 +68,8 @@ stop-development:
 
 # get-protoc will download zip from $2 and install it to $1.
 define get-protoc
-@[ -f $(1) ] || { \
-echo "Downloading $(2) and installing in $(1)" ;\
+@{ \
+echo "Downloading $(2) and installing in $(1)/bin" ;\
 set -e ;\
 TMP_DIR=$$(mktemp -d) ;\
 cd $$TMP_DIR ;\
```

Reply:
I can add that, initially, as per the comment in the PR, I was also extracting the includes in `$(1)/include`...